manager/controllers/app/select_modules.go

```diff
@@ -143,13 +144,13 @@ func (p *PlotterGenerator) validate(item *DataInfo, solution Solution, appContex
 		requiredActions = unsupported
 		// select a cluster for the capability that satisfy cluster restrictions specified in admin config policies
 		if !p.findCluster(item, element, appContext) {
-			p.Log.V(0).Info("Could not find an available cluster for " + moduleCapability.Capability)
+			p.Log.Error().Str(logging.DATASETID, item.Context.DataSetID).Msg("Could not find an available cluster for " + moduleCapability.Capability)
```

Comment:
Debug, not Error. A path is invalidated if a capability along the path can not be deployed.